From c3282532a912dc9188e5804df3821455e61d674e Mon Sep 17 00:00:00 2001
From: Veaceslav Falico <vfalico@redhat.com>
Date: Thu, 21 Aug 2014 11:28:29 -0400
Subject: [ethernet] netxen: Update version to 4.0.82

Message-id: <1408620515-703-5-git-send-email-vfalico@redhat.com>
Patchwork-id: 88517
O-Subject: [PATCH RHEL7 04/10] netxen_nic: Update version to 4.0.82
Bugzilla: 1089356
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

RHBZ#1089356

Testing:
Locally and by QLogic (which supplied the patch)

Description:
Backport of the following commit:

commit fac87a8ecd02a99474d817b6784b6722736f9f0f
Author: Shahed Shaikh <shahed.shaikh@qlogic.com>
Date:   Fri Sep 27 01:42:27 2013 -0400

    netxen_nic: Update version to 4.0.82

    Signed-off-by: Shahed Shaikh <shahed.shaikh@qlogic.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/qlogic/netxen/netxen_nic.h b/drivers/net/ethernet/qlogic/netxen/netxen_nic.h
index e8eff3e..9adcdbb 100644
--- a/drivers/net/ethernet/qlogic/netxen/netxen_nic.h
+++ b/drivers/net/ethernet/qlogic/netxen/netxen_nic.h
@@ -53,8 +53,8 @@
 
 #define _NETXEN_NIC_LINUX_MAJOR 4
 #define _NETXEN_NIC_LINUX_MINOR 0
-#define _NETXEN_NIC_LINUX_SUBVERSION 81
-#define NETXEN_NIC_LINUX_VERSIONID  "4.0.81"
+#define _NETXEN_NIC_LINUX_SUBVERSION 82
+#define NETXEN_NIC_LINUX_VERSIONID  "4.0.82"
 
 #define NETXEN_VERSION_CODE(a, b, c) (((a) << 24) + ((b) << 16) + (c))
 #define _major(v) (((v) >> 24) & 0xff)
-- 
1.7.1